lib/ViewModels/LocationBarViewModel.js

```diff
@@ -26,12 +27,18 @@ var LocationBarViewModel = function(options) {
     this.terria = options.terria;
     this.mapElement = options.mapElement;
     this.geoidModel = defaultValue(options.geoidModel, new EarthGravityModel1996(this.terria.baseUrl + 'data/WW15MGH.DAC'));
+    this.useProjection = defaultValue(options.useProjection, false);
 
     this.latitude = '43.199°S';
     this.longitude = '154.461°E';
     this.elevation = '28m';
+    this.utmZone = '56';
+    this.northing = '5216050.13';
+    this.easting = '618699.50';
+    this.proj4longlat = '+proj=longlat +ellps=WGS84 +datum=WGS84 +units=degrees +no_defs';
+    this.proj4utm = '+proj=utm +ellps=GRS80 +units=m +no_defs';
```

Comment:
Yes, it should be possible to define any projection with a PROJ4 line like that.  I think the problem is mostly terminology...  Easting/Northing/Zone, as well as `proj4utm`.